<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="guessButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #8BCF8B;  /* 更鲜艳的绿色，介于淡绿与亮绿之间 */
    color: #ffffff;
    border: none;
    border-radius: 8px;
    padding: 8px 16px;
    font: 12pt &quot;Microsoft YaHei&quot;;
}
QPushButton:hover {
    background-color: #71C871;  /* 更深一层，突出悬停效果 */
}
QPushButton:pressed {
    background-color: #5EBE5E;  /* 按下时的深绿色 */
}</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultlabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="suggestionlist">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>441</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="historylist">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>701</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="restartButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #42A5F5;
    color: #ffffff;
    border: none;
    border-radius: 8px;
    padding: 8px 16px;
    font: 12pt &quot;Microsoft YaHei&quot;;
}
QPushButton:hover {
    background-color: #1E88E5;
}
QPushButton:pressed {
    background-color: #1976D2;
}</string>
    </property>
    <property name="text">
     <string>重新开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="surrenderButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f44336;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 8px 16px;
    font: 12pt &quot;Microsoft YaHei&quot;;
}
QPushButton:hover {
    background-color: #d32f2f;
}</string>
    </property>
    <property name="text">
     <string>投降</string>
    </property>
   </widget>
   <widget class="QLabel" name="countlabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font: bold 16pt &quot;Microsoft YaHei&quot;;
    color: #333333;
    background-color: #f0f0f0;
    border: 2px solid #aaaaaa;
    border-radius: 10px;
    padding: 5px;
    qproperty-alignment: AlignCenter;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="statsButton">
    <property name="geometry">
     <rect>
      <x>662</x>
      <y>590</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #AB47BC;
    color: #ffffff;
    border: none;
    border-radius: 8px;
    padding: 6px 14px;
    font: 11pt &quot;Microsoft YaHei&quot;;
}
QPushButton:hover {
    background-color: #9C27B0;
}
QPushButton:pressed {
    background-color: #7B1FA2;
}</string>
    </property>
    <property name="text">
     <string>统计数据</string>
    </property>
   </widget>
   <widget class="QWidget" name="titlebar" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QPushButton" name="minimizeButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>×</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
